luatest/server.lua

```diff
@@ -297,6 +412,11 @@ end
 function Server:start(opts)
     checks('table', {wait_until_ready = '?boolean'})
 
+    opts = opts or {}
+    if self.config_file and opts.wait_until_ready == nil then
+        opts.wait_until_ready = self.net_box_uri ~= nil
+    end
```

Comment:
A default value for the `wait_until_ready` variable is determined below, at end of the function. I would coalesce this block with the existing logic.

Reply:
Fixed.